docs/tutorials/signed-provenance-tutorial.md

```diff
@@ -18,6 +18,8 @@ This tutorial will guide you through:
 * Building an image with kaniko in a Tekton TaskRun
 * Verifying the signed image and the signed provenance
 
+![signed-provenance-setup](./images/signed_provenance.png)
```

Comment:
I'm concerned the diagram might be too detailed for people who haven't yet done the tutorial and don't have the context around it yet -- what do you think of putting it at the end of the tutorials so that people have a visual aid for what they just ran through?